api/tests/unit/features/test_unit_features_views.py

```diff
@@ -541,24 +544,47 @@ def test_add_owners_adds_owner(client, project):
     data = {"user_ids": [user_1.id, user_2.id]}
 
     # When
-    json_response = client.post(
-        url, data=json.dumps(data), content_type="application/json"
-    ).json()
+    response = client.post(url, data=json.dumps(data), content_type="application/json")
+    # Then
+    response.status_code == status.HTTP_400_BAD_REQUEST
+    response.data == ["Some users not found"]
```

Comment:
Let's also verify that the users were not added to the feature. Something like 

```python
assert feature.owners.filter(id__in=[user_1.id, user_2.id]).count() == 0
```

Reply:
Good ideas, done!